app/assets/javascripts/admin/addon/routes/admin-schema.js

```diff
@@ -1,11 +1,12 @@
 import Route from "@ember/routing/route";
+import { service } from "@ember/service";
 import SiteSetting from "admin/models/site-setting";
 
-export default class AdminPluginsShowSchemaRoute extends Route {
+export default class AdminSchemaRoute extends Route {
+  @service routeHistory;
+
   async model(params) {
-    const plugin = this.modelFor("adminPlugins.show");
-    const [pluginSettings] = await SiteSetting.findAll({
-      plugin: plugin.id,
+    const [, pluginSettings] = await SiteSetting.findAll({
```

Comment:
Out of curiosity, why are we ignoring the first result? Is the API returning some bad data?

Reply:
<img width="634" alt="Screenshot 2025-05-13 at 13 08 56" src="https://github.com/user-attachments/assets/5563996f-f3c4-465f-8c37-06fe973abdd1" />


without the plugin, it always return the `default_locale` setting(which is weird)

Reply:
Very strange! Perhaps because defaults for settings can be locale-dependent, and so UI for other settings might need it?

I guess it's fine, but could we do something like `.find((s) => s.setting === name)`? That feels less fragile than just picking the second element.

Reply:
I think it can be fine (TGX found to be a [minor](https://github.com/discourse/discourse/pull/32534#discussion_r2078801450)), but now that is only bringing 2 units of data, I guess is fine